core/pubnub_ccore.c

```diff
@@ -385,6 +388,7 @@ enum pubnub_res pbcc_where_now_prep(struct pbcc_context* pb, const char* uuid)
                                 uuid);
     URL_PARAMS_INIT(qparam, PUBNUB_MAX_URL_PARAMS);
     if (pubnub_uname()) { ADD_URL_PARAM(qparam, pnsdk, pubnub_uname()); }
+    if (pb_uuid) { ADD_URL_PARAM(qparam, uuid, pb_uuid); }
```

Comment:
@budgetpreneur can we have here and in other places this line:
```
char const* const uname = pubnub_uname();
if (uname) { ADD_URL_PARAM(qparam, pnsdk, uname); }
```
where, `pnsdk` should be set instead of:
```
if (pubnub_uname()) { ADD_URL_PARAM(qparam, pnsdk, pubnub_uname()); }
```
?

I know, it is not related to scope of this change, just maybe :)

Reply:
ok..made changes per your comment.

Reply:
Thanks!